starrocks/benchmark.sh

```diff
@@ -1,3 +1,60 @@
 #!/bin/bash
 
 # Install
+wget https://download.starrocks.com/en-US/download/request-download/40/StarRocks-2.3.0-rc03.tar.gz
+tar zxvf StarRocks-2.3.0-rc03.tar.gz
+cd StarRocks-2.3.0-rc03/
+
+# Install dependencies
+yum install -y java-1.8.0-openjdk-devel.x86_64
+
+# Create directory for FE and BE
+IPADDR=`hostname -i`
+export STARROCKS_HOME=`pwd`
+mkdir -p meta storage
+
+# Start Frontend
+echo "meta_dir = ${STARROCKS_HOME}/meta " >> fe/conf/fe.conf
+fe/bin/start_fe.sh --daemon
+
+# Start Backend
+echo "storage_root_path = ${STARROCKS_HOME}/storage" >> be/conf/be.conf
+echo "disable_storage_page_cache = false" >> be/conf/be.conf
+echo "storage_page_cache_limit = 4G" >> be/conf/be.conf
+echo "mem_limit=90%" >> be/conf/be.conf
+be/bin/start_be.sh --daemon
+
+# Setup cluster
+mysql -h 127.0.0.1 -P9030 -uroot -e "ALTER SYSTEM ADD BACKEND '${IPADDR}:9050' "
+mysql -h 127.0.0.1 -P9030 -uroot -e "CREATE DATABASE hits "
+mysql -h 127.0.0.1 -P9030 -uroot -e "SET GLOBAL enable_column_expr_predicate=true"
```

Comment:
According to the benchmark methodology, 

> It's better to use the default settings and avoid fine-tuning. Configuration changes can be applied if it is considered strictly necessary and documented.

https://github.com/ClickHouse/ClickBench/blob/main/README.md#installation-and-fine-tuning

Reply:
This is a general optimization for all scenarios, not just for this benchmark. We would turn on it in the next release version and check the benchmark result again.